restx-core-shell/src/main/java/restx/core/shell/DepsShellCommand.java

```diff
@@ -72,11 +72,21 @@ protected String resourceMan() {
         @Override
         public void run(RestxShell shell) throws Exception {
             File mdFile = mdFile(shell);
-            if (!mdFile.exists()) {
+            if(!mdFile.exists()) {
```

Comment:
I prefer the code style using the space between the `if` and the parenthesis, which is the standard code convention: http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-142311.html

But no offense taken if you prefer this style.

Reply:
Will apply it from now, but won't apply it back otherwise it would imply too much noise in `git blame` usages